src/go/rpk/pkg/cli/cmd/api/consume.go

```diff
@@ -68,7 +68,9 @@ func NewConsumeCommand(client func() (sarama.Client, error)) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "consume <topic>",
 		Short: "Consume records from a topic",
-		Args:  cobra.ExactArgs(1),
+		Args:  exactArgs(1, "topic's name is missing."),
+		// We don't want Cobra printing CLI usage help if the error isn't about CLI usage.
+		SilenceUsage: true,
```

Comment:
Because we are using Cobra's `RunE` function, any errors returned from it will make Cobra to automatically print the Usage help, even if the error has nothing to do with CLI usage. We may want to re-structure all the CLI portion after GA, to have more flexibility on its UX.